### PR TITLE
Okpage and api faking implementation

### PR DESCRIPTION
**Pull request by ValantineSuh**
Implemented the Ok Confirmation Page and faked a Send_Money API

### CODE REVIEW

File: power-pay-frontend/src/main.tsx
Comment (stephane-segning) [inline power-pay-frontend/src/main.tsx:1]: Cancel changes made in this file please

**Review by stephane-segning — APPROVED** [review]
Awesome work!

**Review by stephane-segning — CHANGES_REQUESTED** [review]
Please remove unrelated changes

**Comment by ValantineSuh** [comment]
> Awesome work!

Thank you Sir

File: power-pay-frontend/src/main.tsx
Comment (stephane-segning) [inline power-pay-frontend/src/main.tsx:1]: Cancel all changes from this file

File: power-pay-frontend/src/App.css
Comment (stephane-segning) [inline power-pay-frontend/src/App.css:1]: Good. How can you do this using only `tailwindcss`?

File: power-pay-frontend/src/main.tsx
Comment (stephane-segning) [inline power-pay-frontend/src/main.tsx:1]: Cancel all changes from this file

**Review by stephane-segning — CHANGES_REQUESTED** [review]

**Comment by ValantineSuh** [comment]
Hello @stephane-segning  due to the unverified and the partially verified commits on this our branch as you could see, we had to delete the branch but we forgot to cancel the pull request. 
Consequently, this ticket was repeated by @Motouom  and @ValantineSuh  and we made a pull request and it was approved and merged.
- https://github.com/ADORSYS-GIS/e2e-banking-app/pull/110

**Comment by ValantineSuh** [comment]
So sir @stephane-segning, this pull request is to be closed, if it's okay by you.

> Hello @stephane-segning due to the unverified and the partially verified commits on this our branch as you could see, we had to delete the branch but we forgot to cancel the pull request. Consequently, this ticket was repeated by @Motouom and @ValantineSuh and we made a pull request and it was approved and merged.
> 
> * [Implement the frontend page for "ok" using ppa and api faking#27 #110](https://github.com/ADORSYS-GIS/e2e-banking-app/pull/110)

So sir @stephane-segning, this pull request is to be closed, if it's okay by you.

**Pull request closed without merging**